flang/test/Parser/continuation-in-conditional-compilation.f

```diff
@@ -0,0 +1,16 @@
+! RUN: %flang_fc1 -fopenmp -fopenacc -E %s 2>&1 | FileCheck %s
+      program main
+! CHECK: k01=1+1
+      k01=1+
+!$   &  1
+
+! CHECK: !$omp parallel
+!$omp para
+!$omp+llel
```

Comment:
Although I couldn't find anything in OpenMP spec that disallows or explicitly allows breaking a directive name in 2 lines, I'm a bit uncomfortable with testing it. Maybe this could be changed to something like this:
```suggestion
! CHECK: !$omp parallel private(k01)
!$omp parallel
!$omp+ private(k01)
```